modules/tag-input/main/index.coffee

```diff
@@ -116,8 +116,14 @@ class TagInput extends hx.EventEmitter
   add: (name, cssclass) ->
     if hx.isArray(name)
       addTag(this, n, cssclass) for n in name
-    else
+    else if name
       addTag(this, name, cssclass)
+    else
+      hx.consoleWarning(
+        'TagInput.add was passed the wrong argument type',
+        'TagInput.add accepts an array or string argument, you supplied:',
+        name
+      )
```

Comment:
It could but I don't think it should.
Add is for adding a single item to the list whereas items is for adding multiple items to the list.
We could add the ability to add an array of tags but I'm not sure why you'd ever need to do that as opposed to just using `items` to set the list?